faq.md

```diff
@@ -499,6 +499,81 @@ If you are using the Flutter plugin for IntelliJ, you can launch the app in
 profile or release mode using the menu entries **Run > Flutter Run in Profile Mode**
 or **Release Mode**.
 
+### What programming paradigm does Flutter's framework use?
+
+Flutter is a multiparadigm programming environment. Many programming techniques
+developed over the past few decades are used where they are appropriate, to solve
```

Comment:
tweaked the wording to: "Many programming techniques developed over the past few decades are used in Flutter, each to solve a particular set of problems in a clean way."